content/rma_intro.tex

```diff
@@ -2,7 +2,7 @@
 mechanisms of the \openshmem{} \ac{API}. While using these mechanisms, the user
 is required to provide parameters only on the calling side. A characteristic of
 one-sided communication is that it decouples communication from the
-synchronization. One-sided communication mechanisms transfer the data but do not
+synchronization. One-sided communication mechanisms transfer the data, but do not
```

Comment:
Admittedly that comma is a stylistic choice of mine. Many of the sentences within this spec tend to be rather lengthy, so I like to break them up where possible. Technically speaking it isn't improper grammar, but it is unpopular because the comma tends to be overused. If the consensus is that this edit shouldn't make it into the final spec, I will remove it.

What are some of the other instances where you saw flaws in the edits? Again, some of these changes, such as the changing of parentheses to commas, are stylistic choices of mine. I believe that with such edits the document will read better, and that is not to say that the way it was originally written was intrinsically wrong.